src/main/java/com/devcycle/sdk/server/local/api/DVCLocalClient.java

```diff
@@ -239,4 +306,71 @@ private void validateUser(User user) {
   private boolean isValidServerKey(String serverKey) {
     return serverKey.startsWith("server") || serverKey.startsWith("dvc_server");
   }
+
+  private NullableString createNullableString(String value)
+  {
+    return value == null ? NullableString.newBuilder().setIsNull(true).build() : NullableString.newBuilder().setIsNull(false).setValue(value).build();
+  }
+
+  private NullableDouble createNullableDouble(double value)
+  {
+    return !Double.isNaN(value) ? NullableDouble.newBuilder().setIsNull(false).setValue(value).build() : NullableDouble.newBuilder().setIsNull(true).build();
+  }
+
+  private NullableCustomData createNullableCustomData(Object temp)
+  {
+    if (temp == null)
```

Comment:
We should find a Java code style linter to use, not sure if we use this verbose of a `if {} else {}` style anywhere else. But this is usually more easily solved with some type of automatic styling :P

Reply:
I created https://taplytics.atlassian.net/browse/DVC-7415 to investigate adding linter